### PR TITLE
Ft users signup

### PR DESCRIPTION
**Pull request by abibou1**

### CODE REVIEW

**Comment by abibou1** [comment]
I used CSS to center the signup box vertically.

File: package.json [hunk 1]
```diff
@@ -12,15 +12,15 @@
...
     "build:server": "cd server && npm install && npm run build:server",
     "build": "concurrently \"npm run build:client\" \"npm run build:server\""
   },
-  "devDependencies": {
-    "husky": "^4.2.5",
-    "lerna": "^3.22.1"
-  },
   "dependencies": {
+    "husky": "^4.2.5",
+    "lerna": "^3.22.1",
```
Comment (mazma1) [inline package.json:23]: Why did you move these to dependencies?

File: server/package.json [hunk 2]
```diff
@@ -18,7 +18,14 @@
...
     "jsonwebtoken": "^8.5.1",
-    "react-scripts": "3.4.1"
+    "react-scripts": "3.4.1",
+    "eslint": "^6.8.0",
+    "eslint-config-airbnb": "^18.2.0",
+    "eslint-plugin-import": "^2.22.0",
+    "eslint-plugin-jest": "^23.20.0",
+    "eslint-plugin-jsx-a11y": "^6.3.1",
+    "husky": "^4.2.5",
+    "nodemon": "^2.0.4"
```
Comment (mazma1) [inline server/package.json:28]: And these too, why did you move them?

What's the difference between "dependencies" and "devDependencies"?

**Review by mazma1 — COMMENTED** [review]
Just a small observation before we close this out

File: server/package.json (hunk 2 above)
Comment (abibou1) [reply server/package.json]: I moved them when I was solving the jwt problem. I separated them now. Dependencies are the main packages that are essential for production while devDependencies like linting packages help during development.
